camera.py

```diff
@@ -0,0 +1,57 @@
+import requests
+from requests.auth import HTTPDigestAuth
+
+class camera:
+    def __init__(self, ID="", url="", manufacturer="", calendar="", pos=-1, status=0):
+        self.ID = ID
```

Comment:
Since I'm complaining about naming conventions already, variables should be lowercase, see https://peps.python.org/pep-0008/#function-and-variable-names

```suggestion
    def __init__(self, id="", url="", manufacturer="", calendar="", pos=-1, status=0):
        self.id = id
```

Also, I'm not sure if you want an empty string as a default value for `id` or `url`? That doesn't really make sense, does it?